datadog_checks_dev/datadog_checks/dev/tooling/commands/dep.py

```diff
@@ -6,11 +6,11 @@
 import click
 from six import itervalues
 
-from .utils import (
+from .console import (
     CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success, echo_waiting, echo_warning
 )
 from ..constants import get_root, get_agent_requirements
-from ..dep import (
+from ..requirements import (
```

Comment:
Can we please rename this back to some variant of the word `dependency`?

Reply:
that module contains 2 classes that somehow refers the word `package` and a couple of functions that parse requirements files, to me `dependency` is a bit misleading. I'll find something else